__init__.py is not reexporting its members

Once #699 is fixed, we will still have some typing issues because `__init__.py` is not defining `__all__` or reexporting the members. See https://mypy.readthedocs.io/en/stable/command_line.html#cmdoption-mypy-no-implicit-reexport for more details.


Without this, mypy produces the following error:

```
error: Module "django_rq" does not explicitly export attribute "get_queue"  [attr-defined]
```

This can be solved by just adding to `__init__.py`  the following code:

```python
__all__ = [
  "enqueue",
  "get_connection",
  "get_queue",
  "get_scheduler"
  "get_worker",
  "job",
  "VERSION",
]
```

--- a/django_rq/__init__.py
+++ b/django_rq/__init__.py
@@ -6,10 +6,11 @@ from .queues import enqueue, get_queue, get_scheduler
 from .workers import get_worker
 
 __all__ = [
-    "job",
+    "__version__",
     "enqueue",
     "get_connection",
     "get_queue",
     "get_scheduler",
     "get_worker",
+    "job",
 ]

